portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,82 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+var absolute_index = 0;
+//absolute_index is the index of the picture
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function loadIntro(){
+    window.location =("/intro.html");
+}
+
+function getPhotos_pre(){
+    window.location=("./photography.html");
+    getPhotos(0);
+}
+
+function getPhotos(move_dir){
+  
+  if(move_dir==1){
+    //move to next image
+    absolute_index++;
```

Comment:
in our javascript, let's only use CamelCase.

Also, let's use the same spacing and styling we discussed above for java.